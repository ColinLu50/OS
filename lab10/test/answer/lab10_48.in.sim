4566
326
472
261
466
258
