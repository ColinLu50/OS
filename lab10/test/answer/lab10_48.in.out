FCFS
205 -> 234 -> 21 -> 63 -> 6 -> 3 -> 239 -> 165 -> 157 -> 114 -> 147 -> 103 -> 258 -> 260 -> 88 -> 146 -> 101 -> 149 -> 124 -> 46 -> 151 -> 72 -> 28 -> 6 -> 267 -> 19 -> 262 -> 192 -> 264 -> 148 -> 71 -> 260 -> 164 -> 43 -> 211 -> 123 -> 125 -> 43 -> 66 -> 264 -> 260 -> 108 -> 56 -> 225 -> 259 -> 136 -> 166 -> 231 -> 61 -> 139 -> 159
4566
SSTF
205 -> 211 -> 225 -> 231 -> 234 -> 239 -> 258 -> 259 -> 260 -> 260 -> 260 -> 262 -> 264 -> 264 -> 267 -> 192 -> 166 -> 165 -> 164 -> 159 -> 157 -> 151 -> 149 -> 148 -> 147 -> 146 -> 139 -> 136 -> 125 -> 124 -> 123 -> 114 -> 108 -> 103 -> 101 -> 88 -> 72 -> 71 -> 66 -> 63 -> 61 -> 56 -> 46 -> 43 -> 43 -> 28 -> 21 -> 19 -> 6 -> 6 -> 3
326
SCAN
205 -> 192 -> 166 -> 165 -> 164 -> 159 -> 157 -> 151 -> 149 -> 148 -> 147 -> 146 -> 139 -> 136 -> 125 -> 124 -> 123 -> 114 -> 108 -> 103 -> 101 -> 88 -> 72 -> 71 -> 66 -> 63 -> 61 -> 56 -> 46 -> 43 -> 28 -> 21 -> 19 -> 6 -> 3 -> 0 -> 211 -> 225 -> 231 -> 234 -> 239 -> 258 -> 259 -> 260 -> 262 -> 264 -> 267
472
C-SCAN
205 -> 192 -> 166 -> 165 -> 164 -> 159 -> 157 -> 151 -> 149 -> 148 -> 147 -> 146 -> 139 -> 136 -> 125 -> 124 -> 123 -> 114 -> 108 -> 103 -> 101 -> 88 -> 72 -> 71 -> 66 -> 63 -> 61 -> 56 -> 46 -> 43 -> 28 -> 21 -> 19 -> 6 -> 3 -> 0 -> 267 -> 267 -> 264 -> 262 -> 260 -> 259 -> 258 -> 239 -> 234 -> 231 -> 225 -> 211
261
LOOK
205 -> 192 -> 166 -> 165 -> 164 -> 159 -> 157 -> 151 -> 149 -> 148 -> 147 -> 146 -> 139 -> 136 -> 125 -> 124 -> 123 -> 114 -> 108 -> 103 -> 101 -> 88 -> 72 -> 71 -> 66 -> 63 -> 61 -> 56 -> 46 -> 43 -> 28 -> 21 -> 19 -> 6 -> 3 -> 211 -> 225 -> 231 -> 234 -> 239 -> 258 -> 259 -> 260 -> 262 -> 264 -> 267
466
C-LOOK
205 -> 192 -> 166 -> 165 -> 164 -> 159 -> 157 -> 151 -> 149 -> 148 -> 147 -> 146 -> 139 -> 136 -> 125 -> 124 -> 123 -> 114 -> 108 -> 103 -> 101 -> 88 -> 72 -> 71 -> 66 -> 63 -> 61 -> 56 -> 46 -> 43 -> 28 -> 21 -> 19 -> 6 -> 3 -> 267 -> 264 -> 262 -> 260 -> 259 -> 258 -> 239 -> 234 -> 231 -> 225 -> 211
258
